python/serve.py

```
@@ -139,7 +151,17 @@ def main():
           f"seq_len={cfg.seq_len}, vocab={cfg.vocab_size}")
     print(f"  Memory: rule={cfg.memory_rule}, composition={cfg.composition}, k={cfg.k}")
     print(f"  Params: {params.num_params():,}")
+
+    gpu_model = None
+    if args.gpu and hasattr(nl_hecate, "GpuModel"):
+        gpu_model = nl_hecate.GpuModel.from_params(params, cfg)
+        print(f"  Device: GPU")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)